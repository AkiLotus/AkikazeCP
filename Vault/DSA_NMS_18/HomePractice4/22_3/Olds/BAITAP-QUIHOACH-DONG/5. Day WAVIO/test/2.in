20
2 6 5 4 1 2 3 7 8 5 6 2 1 4 5 2 3 0 1 5

