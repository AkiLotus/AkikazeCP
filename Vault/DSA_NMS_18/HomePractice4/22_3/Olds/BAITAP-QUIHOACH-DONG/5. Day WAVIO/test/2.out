9
1 2 3 7 8 6 5 3 1 